android/src/main/java/com/opentokreactnative/OTRNSubscriberPackage.kt

```diff
@@ -27,13 +27,12 @@ class OTRNSubscriberPackage : TurboReactPackage() {
     override fun getReactModuleInfoProvider(): ReactModuleInfoProvider = ReactModuleInfoProvider {
         mapOf(
             OTRNSubscriberManager.REACT_CLASS to ReactModuleInfo(
-                OTRNSubscriberManager.REACT_CLASS, // _name =
-                OTRNSubscriberManager.REACT_CLASS, // _className =
-                false, // _canOverrideExistingModule =
-                false, // _needsEagerInit =
-                true,  // hasConstants
-                false, // isCxxModule =
-                true, // isTurboModule =
+                OTRNSubscriberManager.REACT_CLASS, // name
+                OTRNSubscriberManager.REACT_CLASS, // className
+                false, // canOverrideExistingModule
+                false, // needsEagerInit
+                false, // isCxxModule
+                true, // isTurboModule
```

Comment:
Again, The version of the constructor with the `hasConstants` param is deprecated.